app/controllers/users_controller.rb

```diff
@@ -21,6 +21,10 @@ def edit
 		render partial:'form'
 	end
 
+	def allnames
+		render json: User.all.map {|model| model.tagName }
```

Comment:
consistent spacing: `{ |model| model.tagName }` instead of `{|model| model.tagName }`